fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -2045,6 +2047,13 @@ void SimulationConfig::generateNormalConfig(const TestConfig& testConfig) {
 	setConfigDB(testConfig);
 }
 
+bool validEncryptAndTenantMode(EncryptionAtRestMode encryptMode, TenantMode tenantMode) {
+	if (encryptMode.mode == EncryptionAtRestMode::DISABLED || encryptMode.mode == EncryptionAtRestMode::CLUSTER_AWARE) {
```

Comment:
nit: Please add function documentation OR document above `if` block detailing the motivation of the checks done here.